backend/src/api/masters/masters.api.ts

```diff
@@ -48,6 +48,16 @@ const initMastersApi: FastifyPluginAsync<Options> = async (fastify, opts) => {
       return rep.send(user).status(HttpCode.CREATED);
     },
   });
+  fastify.route({
+    method: HttpMethod.GET,
+    url: MastersApiPath.USER,
+    async handler(req, rep) {
+      const authorization: string = req.headers.authorization ?? '';
```

Comment:
but now you have `Bearer KOKEN` string, ins't it? 😕

Reply:
I get a token in the token service

Reply:
service needs to get token (not a header)